core/src/main/java/nl/nn/adapterframework/processors/InputOutputPipeProcessor.java

```diff
@@ -202,40 +202,40 @@ public PipeRunResult processPipe(PipeLine pipeLine, IPipe pipe, Message message,
 		return super.processPipe(pipeLine, pipe, message, pipeLineSession);
 	}
 
-	private String restoreMovedElements(String invoerString, PipeLineSession pipeLineSession) throws IOException {
+	private String restoreMovedElements(String inputString, PipeLineSession pipeLineSession) throws IOException {
```

Comment:
I thought these changes I also applied in my PR, on the CompactSaxHandler :wink: 
Good at least, that we have the same ideas.

Reply:
Oh I guess you probably did. When I looked at that, I did think "hey didn't I do that already?"; I had this parked in a workspace for some weeks now together with a bunch of other changes and only today pushed it with some more changes that I did today... I already forgot that it was in here.

We'll have to be careful for conflicts now.

Reply:
Yeah, perhaps I should've put those in a separate PR.. but I thought that I could merge that PR already a few days ago 🤔